tools/install_nixl_from_source_ubuntu.py

```diff
@@ -9,6 +9,7 @@
 
 # --- Configuration ---
 WHEELS_CACHE_HOME = os.environ.get("WHEELS_CACHE_HOME", "/tmp/wheels_cache")
+NIXL_VERSION = os.environ.get("NIXL_VERSION", "0.7.0")
```

Comment:
I feel default should always use latest. so we don't need change this file frequently. user/ CI script can set this env vars by themselves. does this make sense?